README.md

```diff
@@ -53,11 +52,9 @@ const
     y = @[1, 2, 3, 4, 5]
 
 var
-    vandermonde: seq[seq[int]]
+    vandermonde: newSeq[seq[int]]()
```

Comment:
If you want to change it to `newSeq` you forgot to replace the `:` by a `=`

Reply:
Good point. Sorry for the mistake. I just fixed it.